src/db/collections/analytics/output_activity.rs

```diff
@@ -41,39 +34,21 @@ pub struct NftActivityAnalyticsResult {
 }
 
 #[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[allow(missing_docs)]
+#[serde(default)]
+pub struct FoundryActivityAnalyticsResult {
+    pub created_count: u64,
+    pub transferred_count: u64,
+    pub destroyed_count: u64,
```

Comment:
I am confused. Didn't we want to represent all counts as `usize` and only values as `u64`?

Reply:
Yes. I think this is fixed in another PR though.